### PR TITLE
enable examples' tests

### PR DESCRIPTION
**Pull request by akosyakov**
I've noticed that electron' tests look very bogus:
- wdio does not seem to be used at all
- especs are working around of `electron-main` to start the server, also only checking that the window appears, regardless where it is showing something 

Could we clean it up? Removed unused wdio configs, rename especs -> specs, write proper tests. With another PR for sure.

### CODE REVIEW

**Comment by akosyakov** [comment]
if nobody minds I am going to merge it tomorrow morning.

File: examples/browser/wdio.conf.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
+// @ts-check 
+const http = require('http');
+const path = require('path');
+
+const theiaConfig = require(path.resolve(__dirname, '.yo-rc.json'))['generator-theia'];
```
Comment (svenefftinge) [inline examples/browser/wdio.conf.js]: shouldn't `.yo-rc.json` be the input for the generator, only

Comment (akosyakov) [inline examples/browser/wdio.conf.js]: I would prefer to start the server with `src-gen/backend/main` and then use its address and port to configure wdio. Unfortunately, it is not possible. As another solution, we can allow passing host and port to `src-gen/backend/main` to override defaults from the generator.

**Comment by epatpol** [comment]
LGTM Indeed at the moment wdio only tests the browser example, I don't remember if there's an easy  way to make it work with electron. Whichever path we end up taking for ui tests, it would be nice not having to write two different ones for electron/chrome.

**Comment by akosyakov** [comment]
@svenefftinge I've removed reading yo config, I will solve it differently in another PR, for now just hard-coded host/port in the wdio config

**Pull request merged**